src/common/logging.hpp

```diff
@@ -40,26 +40,27 @@
 
 #include "common/types.hpp"
 
+#ifndef OTBR_LOG_TAG
+#define OTBR_LOG_TAG ""
+#endif
+
 /**
- * Logging level, which is identical to syslog
+ * Logging level.
  *
  */
-enum
+typedef enum
 {
-    OTBR_LOG_EMERG,   /* system is unusable */
-    OTBR_LOG_ALERT,   /* action must be taken immediately */
-    OTBR_LOG_CRIT,    /* critical conditions */
-    OTBR_LOG_ERR,     /* error conditions */
-    OTBR_LOG_WARNING, /* warning conditions */
-    OTBR_LOG_NOTICE,  /* normal but significant condition */
-    OTBR_LOG_INFO,    /* informational */
-    OTBR_LOG_DEBUG,   /* debug-level messages */
-};
+    OTBR_LOG_LEVEL_CRIT, ///< Critical conditions.
+    OTBR_LOG_LEVEL_WARN, ///< Warning conditions.
+    OTBR_LOG_LEVEL_NOTE, ///< Normal but significant condition.
+    OTBR_LOG_LEVEL_INFO, ///< Informational.
+    OTBR_LOG_LEVEL_DEBG, ///< Debug-level messages.
+} otbrLogLevel;
```

Comment:
This is ot-br-**posix**, I think the logging level defined for POSIX is preferred. https://pubs.opengroup.org/onlinepubs/007904975/basedefs/syslog.h.html.